documentation/index.html.md

```diff
@@ -35,7 +35,9 @@ Learn more about advanced topics:
 
 1. Add the Flow.js required libraries to your project. Flow.js requires the following files:
 	* [`jquery.js`](http://jquery.com): document manipulation, DOM element selection, and event handling used by Flow.js
+		* NOTE: Flow.js requires version 2.x of `jquery.js`.
 	* [`lodash.js`](http://lodash.com): utilities and performance enhancements used by Flow.js; also used in [templating](#templates)
+		* NOTE: Flow.js requires version 2.x of `lodash.js`.
```

Comment:
let me know if there a more specific version of either we should be recommending.